WebCryptoAPI/import_export/okp_importKey.https.any.js

```diff
@@ -126,6 +136,28 @@
         }, "Good parameters: " + keySize.toString() + " bits " + parameterString(format, keyData[format], algorithm, extractable, usages));
     }
 
+    // Test importKey/exportKey "alg" behaviours, alg is ignored upon import and alg is missing for Ed25519 and Ed448 JWK export
+    // https://github.com/WICG/webcrypto-secure-curves/pull/24
+    function testJwkAlgBehaviours(algorithm, keyData, crv, usages) {
+        promise_test(function(test) {
+            return subtle.importKey('jwk', { ...keyData, alg: 'this is ignored' }, algorithm, true, usages).
+            then(function(key) {
+                assert_equals(key.constructor, CryptoKey, "Imported a CryptoKey object");
+
+                return subtle.exportKey('jwk', key).
+                then(function(result) {
+                    assert_equals(Object.keys(result).length, keyData.d ? 6 : 5, "Correct number of JWK members");
+                    assert_equals(result.alg, undefined, 'No JWK "alg" member is present');
```

Comment:
Rather than checking one property not being present, perhaps it might make sense to check that the list of properties/keys matches the expected list? In combination with the length check, that should ensure that there are no extra unexpected properties.

Reply:
We're not enumerating exact properties anywhere else, like this we're checking that alg is not present, which is what the change did, hence explicitly checking it, the attribute count and equalJwk check handle the rest.